### PR TITLE
Code review

### PR DESCRIPTION
**Pull request by christinab12**

### CODE REVIEW

File: napari_organoid_counter/_orgacount.py [hunk 1]
```diff
@@ -163,7 +277,14 @@ def update_bboxes_scores(self, shapes_name, new_bboxes, new_scores, new_ids):
                     self.pred_ids[shapes_name] = new_pred_ids
 
     def update_next_id(self, shapes_name, c=0):
+        """ Updates the next id to append to result dicts. If input c is given then that will be the next id. """
```
Comment (IsraMekki0) [inline napari_organoid_counter/_orgacount.py:280]: There's no check that c doesn't already exist. Is that okay?

File: napari_organoid_counter/_orgacount.py [hunk 2]
```diff
@@ -1,48 +1,124 @@
...
+        cur_confidence: float
+            The confidence threshold of the model
+        cur_min_diam: float
+            The minimum diameter of the organoids
+        transfroms: torchvision.transforms.ToTensor
+            The transformation for converting numpy image to tensor so it can be given as an input to the model
+        model: frcnn
+            The Faster R-CNN model
+        img_scale: list of floats
+            A list holiding the image resolution in x and y
```
Comment (IsraMekki0) [inline napari_organoid_counter/_orgacount.py]: holding*

File: napari_organoid_counter/_orgacount.py [hunk 3]
```diff
@@ -1,48 +1,124 @@
...
+        pred_bboxes: dict
+            Each key will be a set of predictions of the model, either past or current, and values will be the numpy arrays 
+            holding the predicted bounding boxes
+        pred_scores: dict
+            Each key will be a set of predictions of the model abd the values will hold the confidence of the model for each
```
Comment (IsraMekki0) [inline napari_organoid_counter/_orgacount.py]: and* (same for the next 2)

File: napari_organoid_counter/_orgacount.py [hunk 4]
```diff
@@ -92,6 +194,8 @@ def run(self,
         self.next_id[shapes_name] = num_predictions+1
 
     def apply_params(self, shapes_name, confidence, min_diameter_um):
+        """ After resutls have been stored in dict this function will filter the dicts based on the confidence
```
Comment (IsraMekki0) [inline napari_organoid_counter/_orgacount.py]: results

File: napari_organoid_counter/_utils.py [hunk 5]
```diff
@@ -12,6 +15,32 @@
 from torchvision.models.detection.faster_rcnn import FastRCNNPredictor
 from torchvision.ops import nms
 
+from napari_organoid_counter import settings
+
+
+def check_for_local_models_and_add():
```
Comment (IsraMekki0) [inline napari_organoid_counter/_utils.py]: I would rename it `add_local_models`

File: napari_organoid_counter/_utils.py [hunk 6]
```diff
@@ -12,6 +15,32 @@
...
+
+def add_to_dict(filepath):
+    """ Given the full path and name of a model in filepath the model is added to the models dict (see settings)"""
+    filepath = Path(filepath)
+    name = filepath.name
+    stem_name = filepath.stem
+    settings.MODELS[stem_name] = {"filename": name, "source": "local"}
+    return stem_name
+
+def return_is_file(path, filename):
```
Comment (IsraMekki0) [inline napari_organoid_counter/_utils.py:39]: Is this function needed?

**Review by IsraMekki0 — APPROVED** [review]
Looks good to me!

File: napari_organoid_counter/_utils.py (hunk 6 above)
Comment (christinab12) [reply napari_organoid_counter/_utils.py:39]: yes, I also now removed os dependancy from _widget.py. I will add also a function here for os.join and then remove dependency also from _orgacount.py

File: napari_organoid_counter/_orgacount.py (hunk 1 above)
Comment (christinab12) [reply napari_organoid_counter/_orgacount.py:280]: what kind of check do you mean?

Comment (IsraMekki0) [reply napari_organoid_counter/_orgacount.py:280]: `self.next_id[shapes_name] = c` Like what if it's already c (or less) is that okay?

**Pull request merged**